homeassistant/components/light/hyperion.py

```diff
@@ -68,15 +70,25 @@ def turn_off(self, **kwargs):
 
     def update(self):
         """Ping the remote."""
-        # just see if the remote port is open
+        # Check if it's up, and showing color
         self._is_available = self.json_request()
+        response = self.json_request({"command": "serverinfo"})
```

Comment:
If you define `_is_on`, why would you still request `_is_available` too ? Why is this new method better ?